### DISCUSSION TITLE
Allow access to files in root folder but restrict files in specific folders

### DISCUSSION THREAD

**Discussion by NecoHorne** [directus/directus#8638] [Archive]
Is it possible to allow public access to the files in the root folder but restrict access to specific folders? 

I have tried adding this to the public -> files - > read but no luck. 

Folder not equal to the folder I am trying restrict

```
{
	"folder": {
		"_neq": "feb09e61-2e7c-407a-9816-cbba17a1d19c"
	}
}
```

![image](https://user-images.githubusercontent.com/30078491/136528522-cf361d16-1d08-42c2-a184-e0f9b21e56b9.png)

**Comment by rijkvanzanten** [comment]
> Is it possible to allow public access to the files in the root folder but restrict access to specific folders?

Should be! The root folder is `null`, so you should be able to use "Folder is null"
